pkg/cmd/cli/cmd/newapp.go

```diff
@@ -132,11 +132,60 @@ type NewAppOptions struct {
 	LogsForObject LogsForObjectFunc
 }
 
+type NewAppOptions struct {
+	AppOptions *BaseOptions
```

Comment:
call this BaseOptions, or embed it anonymously.  Having a NewAppOptions.AppOptions doesn't make sense (and doesn't really reflect what the field is).